sanic/blueprints.py

```diff
@@ -449,16 +449,11 @@ def event(self, event: str, timeout: Optional[Union[int, float]] = None):
 
     @staticmethod
     def _extract_value(*values):
-        value = values[-1]
-        for v in values:
-            if v is not None:
-                value = v
-                break
-        return value
+        return next((v for v in values if v is not None), values[-1])
 
     @staticmethod
     def register_futures(
         apps: Set[Sanic], bp: Blueprint, futures: Sequence[Tuple[Any, ...]]
     ):
         for app in apps:
-            app._future_registry.update(set((bp, item) for item in futures))
+            app._future_registry.update({(bp, item) for item in futures})
```

Comment:
What is the purpose of this change?

Reply:
no need to use set() here as it's slower than just {}